qcodes/instrument/instrument_meta.py

```diff
@@ -0,0 +1,44 @@
+from abc import ABCMeta
+from typing import Any
+
+
+class InstrumentMeta(ABCMeta):
```

Comment:
moving this class into its own module makes sense on its own :)

Reply:
charset-normalizer